<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCell</class>
 <widget class="QWidget" name="frmCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lbHeader">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>WRITE SOMETHING HERE LATER &lt;---</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="pushNew">
         <property name="toolTip">
          <string>Create new record</string>
         </property>
         <property name="statusTip">
          <string>Create new record</string>
         </property>
         <property name="whatsThis">
          <string>Creates new record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/filenew.png</normaloff>:/app_new/filenew.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Setup temporary frame changes</string>
         </property>
         <property name="statusTip">
          <string>Setup temporary frame changes</string>
         </property>
         <property name="whatsThis">
          <string>Setups temporary frame changes</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupDetails">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Landing Site</string>
          </property>
          <property name="buddy">
           <cstring>cmbLS</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLS"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Start Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomTimeCtrl" name="customDtStart"/>
        </item>
       </layout>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>End Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomTimeCtrl" name="customDtEnd"/>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>No of Vessels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Active Vessels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Inactive Vessels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Vessels from Outside</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Estimated</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinNE"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinAE"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinIE"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinOE"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Calculated</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinNC">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinAC">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinIC">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinOC">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Input Details</string>
        </property>
        <property name="buddy">
         <cstring>toolButton</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPlainTextEdit" name="textComments"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Comments</string>
        </property>
        <property name="buddy">
         <cstring>textComments</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushPrevious">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTimeCtrl</class>
   <extends>QWidget</extends>
   <header>customtimectrl.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>pushNew</tabstop>
  <tabstop>cmbLS</tabstop>
  <tabstop>spinNE</tabstop>
  <tabstop>spinNC</tabstop>
  <tabstop>spinAE</tabstop>
  <tabstop>spinAC</tabstop>
  <tabstop>spinIE</tabstop>
  <tabstop>spinIC</tabstop>
  <tabstop>spinOE</tabstop>
  <tabstop>spinOC</tabstop>
  <tabstop>textComments</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pushPrevious</tabstop>
  <tabstop>pushNext</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>frmCell</receiver>
   <slot>createRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>frmCell</receiver>
   <slot>onButtonClick(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>frmCell</receiver>
   <slot>previewRow(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushNext</sender>
   <signal>clicked()</signal>
   <receiver>frmCell</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createRecord()</slot>
  <slot>onButtonClick(QAbstractButton*)</slot>
  <slot>previewRow(QModelIndex)</slot>
  <slot>next()</slot>
 </slots>
</ui>
